common/lib/Support/ForkPiped.cpp

```diff
@@ -40,6 +40,6 @@ int nixd::forkPiped(int &In, int &Out, int &Err) {
 
   In = PipeIn[WRITE];
   Out = PipeOut[READ];
-  Err = PipeOut[READ];
+  Err = PipeErr[READ];
```

Comment:
double-check if it's still needed

Reply:
Confirmed it's not required. Dropping.

Reply:
@inclyc though this change doesn't belong to this PR, I wonder if it's still a legit issue to patch. Thoughts?